### ISSUE TITLE
Install HowTo

### ISSUE THREAD

**Issue by rxa254** [kevinkuns/pytickle#5]
install instructions in the REAMDME?

maybe a setup.py file; idk how annoying it is to put in pip or conda

**Comment by rxa254** [comment]
I've seen that there's even ways to directly install from the repo without checkout (besides pip and conda). Also maybe tag some versions? We may be on pytickle 0.9 right now and can move it to 1.0 once a few users have used it for a month or so.

**Comment by kevinkuns** [comment]
I don't know how to automate setting up the matlab engine. There is a setup section in the README with a link to detailed instructions on setting up the matlab engine.

**Comment by kevinkuns** [comment]
I'll look into installing without checkout.

**Comment by rxa254** [comment]
that's good too, but perhaps you can add something about how you have to define a ENV variable in the shell and how to add the path in the notebook. Or you let the users do the matlab part themselves and then do the rest with 'python setup.py install'

**Comment by rxa254** [comment]
when I run the examples, I get this error:
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-77f9d6678eb2> in <module>
      1 import numpy as np
----> 2 import pytickle.finesse as fin
      3 import pykat
      4 from pytickle.plotting import plotTF
      5 import matplotlib as mpl

ModuleNotFoundError: No module named 'pytickle'
```
are we supposed to setup the path externally from this notebook? I'm not sure how its supposed to find pytickle.

**Comment by kevinkuns** [comment]
Yeah, pytickle has to be in the python path. I usually do that in bashrc or a conda environment's env_vars.sh.

I'll try to get get these installation kinks worked out by next week.

**Comment by rxa254** [comment]
It seems that now there is a PIP method to install.
